### PR TITLE
RFID linked tickets can't be transferred.

### LINKED ISSUE #239: If a Ticket has been linked to RFID, make it unavailable for transfer

### PR DESCRIPTION
**Pull request by bvanderdrift**
Fixes #239

### CODE REVIEW

File: src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java [hunk 1]
```diff
@@ -45,6 +46,11 @@ public TicketServiceImpl(TicketRepository ticketRepository, UserService userServ
         this.teamService = teamService;
     }
 
+    @Autowired
+    public void setRFIDService(RFIDService rfidService){
```
Comment (bvanderdrift) [inline src/main/java/ch/wisv/areafiftylan/service/TicketServiceImpl.java]: This had to be implemented due to a circular dependency between RFIDService and TicketService. Ticket Transfer is dependent on RFID, RFID is dependent on tickets. A solution to remove the circular dependency would be to extract the Ticket Transfer logic to a seperate service. I invite you to continue the discussion in #250.

See [StackOverflow](http://stackoverflow.com/questions/5632596/understanding-spring-context-initialization-order) for motivation.

**Comment by bvanderdrift** [comment]
Please do NOT merge yet. I haven't implemented the case: open-transfer --> link ticket --> perform transfer yet.

**Comment by bvanderdrift** [comment]
Allright, ready to go!

**Comment by martijn9612** [comment]
Looks good!

**Pull request merged**